app/helpers/dynamodb.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, table_name, bucket_name, endpoint_url, table_region):
         self.bucket_name = bucket_name
         self.endpoint = endpoint_url
 
-    def save_item(self, kml_id, kml_admin_id, file_key, timestamp, empty=False):
+    def save_item(self, kml_id, kml_admin_id, file_key, timestamp, empty=False, author=''):
```

Comment:
see comment in other PR: don't you wanna use 'mapviewer' or sth as default? Or would you add this in the viewer then?

Reply:
See comment https://github.com/geoadmin/doc-api-specs/pull/15#discussion_r713771088

Reply:
Changed default to `unknown`